web/packages/core/src/config.ts

```diff
@@ -44,4 +44,5 @@ export const DEFAULT_CONFIG: Required<BaseLoadOptions> = {
     openUrlMode: OpenURLMode.Allow,
     allowNetworking: NetworkingAccessMode.All,
     openInNewTab: null,
+    wsProxy: [],
```

Comment:
I do not like the name of the config option, but I couldn't think of a better name, so suggestions are welcome.

Reply:
I would call it `socketProxy`!

Reply:
Renamed.